tests/jobs/test_scheduler_job.py

```diff
@@ -293,11 +295,7 @@ def test_execute_task_instances_is_paused_wont_execute(self):
             max_active_tasks=dag.max_active_tasks,
             has_task_concurrency_limits=False,
         )
-        dr1 = dag.create_dagrun(
-            run_type=DagRunType.BACKFILL_JOB,
-            execution_date=DEFAULT_DATE,
-            state=State.RUNNING,
-        )
+        dr1 = dag_maker.dag_run
```

Comment:
Thinking about my comment about "two" factories, maybe _this_ is how we could have it in one.

```
dag_maker.make_dag_run(dag_run_args_go="here")
```

That way args to `dag_maker()` are _only_ for the dag, and the DagRun args are separate here.

And for simplicity/common case, we can have 

```
@property
def dag_run(self):
    return self.make_dag_run()
```